Tests/SwiftFormatPrettyPrintTests/FunctionTypeTests.swift

```diff
@@ -0,0 +1,64 @@
+public class FunctionTypeTests: PrettyPrintTestCase {
+  public func testFunctionType() {
+    let input =
+      """
+      func f(g: (_ somevalue: Int) -> String?) {
+        let a = 123
+        let b = "abc"
+      }
+      func f(g: (name currentLevel: Int) -> String?) {
```

Comment:
I don't think the parser allows this; if I put this into the compiler it errors out saying that closure arguments can't have labels, so I think we can remove this example.